### ISSUE TITLE
Optimización, buenas prácticas y “how to” varios con la API JS 4.x

### ISSUE THREAD

**Issue by hhkaos** [Geo-Developers/talks#22]
A petición de [Héctor](https://www.meetup.com/es-ES/Geo-Developers/members/197299632/)

**Comment by sanmolhec** [comment]
como no podía ser de otra forma yo me apunto. 
+1

**Comment by ghost** [comment]
Tambien interesado.
+otro

**Comment by mikelaaf** [comment]
Cuenta con otro.

**Comment by fvara** [comment]
Yo también me apunto. Gracias.

**Comment by carmenicuadrado** [comment]
Me apunto!!

**Comment by hhkaos** [comment]
@ntkog le damos caña y le ponemos fecha a esta charla?

**Comment by ntkog** [comment]
Oído cocina!  Me preparo material  y hacemos una ! Es mi tema favorito , Javascript , jejejejeje

**Comment by fvara** [comment]
En caso de que salga adelante... ¿es posible que sea grabado para los que vivimos "overseas"?
Gracias.

**Comment by ntkog** [comment]
Of course! Lo que hagamos en Meetup, en el 90% de los casos será online y se graba, con lo que podréis disfrutar de ello en el momento que os venga mejor. :-)

**Comment by fvara** [comment]
¡Genial gracias!
